tests/integration/annotation_import/test_label_import.py

```diff
@@ -0,0 +1,83 @@
+- Then with unit tests we can check the types of errors raised
+
+"""
+
+
+def test_create_from_url(client, project, annotation_import_test_helpers):
+    name = str(uuid.uuid4())
+    url = "https://storage.googleapis.com/labelbox-public-bucket/predictions_test_v2.ndjson"
+    label_import = LabelImport.create_from_url(client=client, project_id=project.uid, name=name, url=url)
+    assert label_import.parent_id == project.uid 
+    annotation_import_test_helpers.check_running_state(label_import, name, url)
+
+
+def test_create_from_objects(client, project, object_predictions, annotation_import_test_helpers):
+    name = str(uuid.uuid4())
+
+    label_import = LabelImport.create_from_objects(client=client, project_id=project.uid, name=name, labels=object_predictions)
+    assert label_import.parent_id == project.uid 
+    annotation_import_test_helpers.check_running_state(label_import, name)
+    annotation_import_test_helpers.assert_file_content(label_import.input_file_url, object_predictions)
+
+
+#   TODO: add me when we add this ability
+# def test_create_from_local_file(client, tmp_path, project,
+#                                 object_predictions, annotation_import_test_helpers):
+#     name = str(uuid.uuid4())
+#     file_name = f"{name}.ndjson"
+#     file_path = tmp_path / file_name
+#     with file_path.open("w") as f:
+#         ndjson.dump(object_predictions, f)
+
+#     label_import = LabelImport.create_from_url(client=client, project_id=project.uid, name=name, url=str(file_path))
+
+#     assert label_import.parent_id == project.uid
+#     annotation_import_test_helpers.check_running_state(label_import, name)
+#     annotation_import_test_helpers.assert_file_content(label_import.input_file_url, object_predictions)
+
+
+def test_get(client, project, annotation_import_test_helpers):
+    name = str(uuid.uuid4())
+    url = "https://storage.googleapis.com/labelbox-public-bucket/predictions_test_v2.ndjson"
+
+    label_import = LabelImport.create_from_url(client=client, project_id=project.uid, name=name, url=url)
+
+    assert label_import.parent_id == project.uid
+    annotation_import_test_helpers.check_running_state(label_import, name, url)
+
+
+@pytest.mark.slow
+def test_wait_till_done(client, project, model_run_predictions):
+    name = str(uuid.uuid4())
+    label_import = LabelImport.create_from_objects(client=client, project_id=project.uid, name=name, labels=model_run_predictions)
+
+    assert len(label_import.inputs) == len(model_run_predictions)
+    label_import.wait_until_done()
+    # TODO(grant): some of this is commented out
+    # TODO(grant): since the pipeline is not complete, you will get a failed status
+
+    # assert label_import.state == AnnotationImportState.FINISHED
+    # # Check that the status files are being returned as expected
+    # assert len(label_import.errors) == 0
+    assert len(label_import.inputs) == len(model_run_predictions)
```

Comment:
I use `predictions` from MEA since they're the same thing... maybe in this PR i should take the time to rename this stuff, too?

Reply:
yeah maybe rename that to something more meaningful. Also why aren't you using the `predictions` fixture. https://github.com/Labelbox/labelbox-python/blob/e89b9aa1e8eff8db02a56cc6018f804f0533df24/tests/integration/mal_and_mea/conftest.py#L295

It will have greater coverage of tools.